### PR TITLE
Refactored hyperlinks in topbar

### LINKED ISSUE #206: Change position of channel hyperlink on topbar
The position of `channel` hyperlink in the topbar is uneven in comparison to that of the `Fragment` logo. It can either be brought a little down to make it even or can removed to the right beside the `Get fedora` link.

Screenshot from 2021-11-01 23-55-58

Also, is `Channel` hyperlink on the topbar necessary, when the same functionality is achieved through the `Explore` section? Could you please elaborate @darknao @t0xic0der

### PR DESCRIPTION
**Pull request by subhangi2731**
* Removed the `Channel` hyperlink from the top bar as suggested by @darknao in issue #206 
* Aligned the position of the `Get Fedora` hyperlink which is now perfectly in line with the `Fragment` logo.

Closes #206 

@darknao @t0xic0der Please review

### CODE REVIEW

**Comment by darknao** [comment]
Looks good, but other templates also need this alignment fix.

**Comment by subhangi2731** [comment]
All templates have been updated.

File: src/mote/templates/statfile.html [hunk 1]
```diff
@@ -99,7 +99,7 @@
                             Akashdeep Dhar
                         </a>
                         and
-                        <a style="color:#676767; text-decoration:none;"
+                        <a style="color:#676767; text-decoration:none; margin-top: 4px;"
```
Comment (darknao) [inline src/mote/templates/statfile.html:102]: That's the wrong place.

Comment (subhangi2731) [reply src/mote/templates/statfile.html:102]: Fixed the bug now.

**Review by darknao — APPROVED** [review]
Looks good.

**Comment by subhangi2731** [comment]
@t0xic0der this can be merged too?

File: src/mote/templates/e404page.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
                     <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                     </ul>
                     <div class="d-flex">
-                        <a class="text-white" href="https://getfedora.org/" target="_blank" type="button">Get Fedora</a>
+                        <a class="text-white; margin-top: 4px;" href="https://getfedora.org/" target="_blank" type="button">Get Fedora</a>
```
Comment (t0xic0der) [inline src/mote/templates/e404page.html:34]:
```suggestion
                        <a class="text-white;" style="margin-top: 4px;" href="https://getfedora.org/" target="_blank" type="button">Get Fedora</a>
```

**Review by t0xic0der — CHANGES_REQUESTED** [review]

File: src/mote/templates/e404page.html (hunk 2 above)
Comment (t0xic0der) [inline src/mote/templates/e404page.html:34]:
```suggestion
                        <a class="text-white;" style="margin-top: 4px;" href="https://getfedora.org/" target="_blank" type="button">Get Fedora</a>
```

**Review by t0xic0der — CHANGES_REQUESTED** [review]
To https://github.com/fedora-infra/mote/pull/207#issuecomment-982656347, no.

**Comment by subhangi2731** [comment]
@t0xic0der please check and let me know

**Comment by t0xic0der** [comment]
I think that there's a change in direction for this pull request, after the review discussion that we had with @mairin in our weekly Websites and Apps Team meeting on 30th November 2021. The things that we need to do include - 

1. Removing the buttons from the center of the page
2. Placing them as link on the navbar - pretty much like how the `Channels` button was there

We can either make use of the same pull request or close this one to do stuff in a new one. Your call, @subhangi2731.

**Comment by subhangi2731** [comment]
@t0xic0der I was thinking maybe a new one will be better to make these new changes??

**Comment by subhangi2731** [comment]
@t0xic0der In reference to your comment you meant that the buttons `recent `, `explore` and `lookup`, I should remove from their original place(center) and keep it as link button in navbar like `channels` right? So as I have removed the `channels` hyperlink so in place of the `channels` hyperlink there will be these three buttons `recent`, `explore`, and `lookup` in place of that right with the same name right? Correct me please if I am wrong and then accordingly I will send a new pr

**Comment by t0xic0der** [comment]
@subhangi2731, "all" buttons have to be removed from the center of the page and placed on the top-side as links on the navbar.

**Comment by subhangi2731** [comment]
@t0xic0der sure I will do this

**Comment by t0xic0der** [comment]
I believe this can be closed in the wake of #230 being merged. Please feel free to reply back and reopen the PR if it's otherwise.

**Pull request closed without merging**